selectors-4/Overview.bs

```diff
@@ -2403,23 +2411,63 @@ Resource State Pseudos</h2>
 	particularly images/videos,
 	and allow authors to select them based on some quality of their state.
 
-<h3 id="video-state">
-Video/Audio Play State: the '':playing'' and '':paused'' pseudo-classes</h3>
+<h3 id="video-state"> <!-- keeping this ID because Cool URIs Don't Change -->
```

Comment:
Good point, but no need to explain your ID choices. ^_^

Reply:
Removed in 6599d0b.